src/main/scala/devices/gpio/GPIO.scala

```diff
@@ -17,7 +17,7 @@ import freechips.rocketchip.diplomaticobjectmodel.DiplomaticObjectModelAddressin
 import freechips.rocketchip.diplomaticobjectmodel.model.{OMComponent, OMRegister}
 import freechips.rocketchip.diplomaticobjectmodel.logicaltree.{LogicalModuleTree, LogicalTreeNode}
 
-import sifive.blocks.devices.pinctrl.{PinCtrl, Pin, BasePin, EnhancedPin, EnhancedPinCtrl}
+import sifive.blocks.devices.pinctrl.{LN14LPPEnhancedPin, LN14LPPEnhancedPinCtrl}
```

Comment:
We shouldn't include process-specific information in this repository

Reply:
I am not not sure whether those IOs are process specific or not. It will be better to be able to add pins some where else.